vars/piperPipelineStageInit.groovy

```diff
@@ -70,7 +70,12 @@ import static com.sap.piper.Prerequisites.checkScript
      * Sets the build version.
      * @possibleValues `true`, `false`
      */
-    'artifactPrepareVersion'
+    'artifactPrepareVersion',
+    /**
+     * Retrieve transport request from git commit history.
+     * @possibleValues `true`, `false`
+     */
+    'transportRequestReqIDFromGit'
```

Comment:
~better to rename this variable for example as `containsTransportRequestReqIDFromGit` since it holds a boolean value~.

Reply:
Basically, we do not need this parameter definition here at all, as, if the step config for the step `transportRequestReqIDFromGit` exists, then execution of this step will be eventually done.

Reply:
We need this parameter configured here. 
It will be used [here](https://github.com/SAP/jenkins-library/blob/2536963a053a12ea46de3563ecaa5726bbf968fc/vars/piperPipelineStageInit.groovy#L171).

Reply:
Can you please explain what do you mean here ? 
_**if the step config for the step transportRequestReqIDFromGit exists**_

Reply:
Unfortunately, we cannot change this either at the moment since it needs adaptions in marshaller in api service.